test/index.test.js

```diff
@@ -821,6 +821,18 @@ describe('Optimizely', function() {
           revenue: 999
         }]);
       });
+
+      describe('the Optimizely X Fullstack JavaScript client is present', function() {
+        beforeEach(function() {
+          window.optimizelyClientInstance = {};
+          analytics.stub(window.optimizelyClientInstance, 'track');
+        });
+
+        it('should send an event through the Optimizely X Fullstack JS SDK', function() {
+          analytics.track('event', { userId: 'user1', revenue: 9.99, property: 'foo' });
```

Comment:
You should add before this line,

```
analytics.identify('userId');
```

That way this line of code can just be:

```
analytics.track('event', { revenue: 9.99, property: 'foo' });
```

Which is more in line with what Segment customers are actually calling. Most people do not add userId again in their properties since it's already in the payload if you had called `.identify()` prior.